scripts/install-aica-docker.sh

```diff
@@ -1,10 +1,56 @@
 #!/bin/bash
 
+function get_shell_rc_path () {
+  local user home_dir
+  user=$(logname)
+  home_dir=$(eval echo "~${user}")
+  if [[ "${SHELL##*|}" == *"zsh"* ]]; then
+    echo "${home_dir}/.zshrc"
+  elif [[ "${SHELL##*|}" == *"bash"* ]]; then
+    if [[ "$OSTYPE" != "darwin"* ]]; then
+      echo "${home_dir}/.bashrc"
+    else
+      echo "${home_dir}/.bashrc"
+    fi
+  else
+    echo "Currently, only zsh and bash shells are supported for autocompletion with aica-docker."
+    exit 1
+  fi
+}
+
+function source_completion_script () {
+  if grep -Rq "/src/aica-docker-completion.sh" "$1"; then
+    echo "Updating auto completion in $1"
+    grep -v /src/aica-docker-completion.sh "$1" > tmpfile && mv tmpfile "$1"
+  else
+    echo "Writing auto completion to $1"
+  fi
+  echo "source ${SCRIPT_DIR}/src/aica-docker-completion.sh" >> "$1"
+}
+
 if [[ "$OSTYPE" != "darwin"* ]]; then
   mkdir -p /usr/local/bin
 fi
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"/aica-docker.sh
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+SYMLINK="/usr/local/bin/aica-docker"
+
+if test -f "$SYMLINK"; then
+  echo "Updating symbolic link from ${SYMLINK} to ${SCRIPT_DIR}/aica-docker.sh;"
+  sudo rm "${SYMLINK}" || exit 1
+else
+  echo "Creating a symbolic link from ${SYMLINK} to ${SCRIPT_DIR}/aica-docker.sh;"
+fi
+sudo ln -s "${SCRIPT_DIR}/aica-docker.sh" ${SYMLINK}
 
-echo "Creating a symbolic link from /usr/local/bin/aica-docker to ${SCRIPT_DIR}"
-ln -s "${SCRIPT_DIR}" /usr/local/bin/aica-docker
+SHELL_RC_PATH=$(get_shell_rc_path)
+while true; do
+  read -r -p "Do you with to install auto completion for aica-docker to '${SHELL_RC_PATH}'? [YES|no]
+>>>" yn
+  case $yn in
+    "") source_completion_script "${SHELL_RC_PATH}"; break;;
+    yes) source_completion_script "${SHELL_RC_PATH}"; break;;
```

Comment:
you should be able to combine this as follows (but please test it first):
```suggestion
    "" | yes) source_completion_script "${SHELL_RC_PATH}"; break;;
```

Reply:
that does work :)